code-challenges.test.js

```diff
@@ -56,4 +158,53 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("firstVowel", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+        const vowelTester1 = "learn"
+        const vowelTester1Output = 1
+        const vowelTester2 = "academy"
+        const vowelTester2Output = 0
+        const vowelTester3 = "challenges"
+        const vowelTester3Output = 1
+        expect(vowelTester1).toEqual(vowelTester1Output)
+        expect(vowelTester2).toEqual(vowelTester2Output)
+        expect(vowelTester3).toEqual(vowelTester3Output)
+    })
+})
+
 // b) Create the function that makes the test pass.
+    // create a function called vowel 
+    // one parameter = string
+    // using a for loop, 
+        // if there is a vowel in the string 
+            // push string by using the method indexof to find the position of the first vowels
```

Comment:
Since you are using a for loop, you have access automatically to the index of the current iteration...which hint hint, is just what you need in newString.  You then just need the first element from newString.